src/DssSpell.sol

```diff
@@ -53,163 +50,30 @@ contract DssSpellAction is DssAction {
-
-        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
-        D3MInstance memory d3m = D3MInstance({
-            plan:   MORPHO_D3M_PLAN,
-            pool:   MORPHO_D3M_POOL,
-            oracle: MORPHO_D3M_ORACLE
-        });
-        D3MCommonConfig memory cfg = D3MCommonConfig({
-            hub:         D3M_HUB,
-            mom:         D3M_MOM,
-            ilk:         "DIRECT-SPARK-MORPHO-DAI",
-            existingIlk: false,
-            maxLine:     100 * MILLION * RAD, // Set line to 100 million DAI
-            gap:         100 * MILLION * RAD, // Set gap to 100 million DAI
-            ttl:         24 hours,            // Set ttl to 24 hours
-            tau:         7 days               // Set tau to 7 days
-        });
-        D3M4626PoolConfig memory erc4626Cfg = D3M4626PoolConfig({
-            vault: SPARK_MORPHO_VAULT
-        });
-        D3MOperatorPlanConfig memory operatorCfg = D3MOperatorPlanConfig({
-            operator: MORPHO_D3M_OPERATOR
-        });
-        D3MInit.initCommon({
-            dss:     dss,
-            d3m:     d3m,
-            cfg:     cfg
-        });
-        D3MInit.init4626Pool(
-            dss,
-            d3m,
-            cfg,
-            erc4626Cfg
-        );
-        D3MInit.initOperatorPlan(
-            d3m,
-            operatorCfg
-        );
-
-        // Additional actions:
-        // Expand DIRECT_MOM breaker to also include Morpho D3M
-        // Note: this is already done within D3MInit.sol line 209
-
-        // ---------- DSR Change ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-11-under-sta-article-3-3/23910
-
-        // DSR: Decrease the Dai Savings Rate by 2 percentage points from 15.00% to 13.00%
-        DssExecLib.setDSR(THIRTEEN_PCT_RATE, /* doDrip = */ true);
-
-        // ---------- SBE Parameter Updates ----------
-        // Forum: https://forum.makerdao.com/t/smart-burn-engine-the-rate-of-mkr-accumulation-reconfiguration-and-transaction-analysis-parameter-reconfiguration-update-6/23888
-
-        // Decrease the hop parameter for 7,884 seconds from 19,710 seconds to 11,826 seconds.
-        DssExecLib.setValue(MCD_FLAP, "hop", 11_826);
-
-        // Increase the bump parameter for 25,000 DAI from 50,000 DAI to 75,000 DAI.
-        DssExecLib.setValue(MCD_VOW, "bump", 75 * THOUSAND * RAD);
-
-        // ---------- Approve HVBank (RWA009-A) Dao Resolution ----------
-        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/24
-
-        // Approve HVBank (RWA009-A) Dao Resolution with IPFS hash QmStrc9kMCmgzh2EVunjJkPsJLhsVRYyrNFBXBbJAJMrrf
-        // Note: see `dao_resolutions` variable declared above
+        // ---------- Increase the GSM Pause Delay ----------
+        // Forum: TODO
+        // Poll: TODO
+
+        // ----- Increase the GSM Pause Delay by 14 hours, from 16 hours to 30 hours -----
```

Comment:
Following the [reviewer checklist](https://github.com/makerdao/pe-checklists/blob/45b65c3829e66713e84af2ce89c2209df6e5c4be/spell/spell-crafter-mainnet-workflow.md?plain=1#L37), instruction text doesn't need to be wrapped with `-`
```suggestion
        // Increase the GSM Pause Delay by 14 hours, from 16 hours to 30 hours
```

Reply:
Fixed.